extensions/wikia/MonetizationModule/MonetizationModuleHelper.class.php

```diff
@@ -87,28 +86,23 @@ public static function canShowModule() {
 	 * @return string - wiki vertical
 	 */
 	public function getWikiVertical() {
-		wfProfileIn( __METHOD__ );
-
 		$verticalId = WikiFactoryHub::getInstance()->getVerticalId( $this->wg->CityId );
 		if ( empty( self::$verticals[$verticalId] ) ) {
 			$verticalId = WikiFactoryHub::VERTICAL_ID_OTHER;
 		}
 
 		$name = self::$verticals[$verticalId];
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Immediately return this expression instead of assigning it to the temporary variable "$name". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1488)